lambda/main.go

```diff
@@ -61,12 +61,12 @@ func HandleRequest(ctx context.Context) error {
 	err = uploadFileToS3(session, EXPLOITDB_DIR)
 	if err != nil {
 		log.Printf("error while uploading exploit db file to s3. Bucket:%s, Region:%s,Error:%s", AWS_S3_BUCKET, AWS_S3_REGION, err)
-		return error
+		return err
 	}
 
 	// Invalidate CloudFront cache for the uploaded file
-	err := invalidateCloudFrontCache(session)
-	if err != nil {
+	errcf := invalidateCloudFrontCache(session)
```

Comment:
This can still be `err = invalidateCloudFrontCache(session)` Value will automatically be overridden. New variable is not required.

Reply:
okay @namandf

Reply:
updated @namandf